### PR TITLE
Update the ReadMe file

### PR DESCRIPTION
**Pull request by IgorCATech**
Add the "edit" and "delete" profile functions and SCS plug-in description to the doc.

Signed-off-by: Igor Kazmyr <43951184+IgorCATech@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
 
 The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes. Working with data sets and USS files from VSC can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
 
-* Enables developers to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
-* Enables developers to create, modify, rename and upload USS files directly to a z/OS mainframe.
+* Enables you to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
+* Enables you to create, modify, rename and upload USS files directly to a z/OS mainframe.
 * Provides a more streamlined way to access data sets, uss files and jobs.
-* Lets you create and use Zowe CLI `zosmf` compatible profiles.
+* Lets you create, edit and use Zowe CLI `zosmf` compatible profiles.
```
Comment (zFernand0) [inline README.md:13]: Should we also mention "delete" here?

File: README.md [hunk 2]
```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
...
 The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes. Working with data sets and USS files from VSC can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
 
-* Enables developers to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
-* Enables developers to create, modify, rename and upload USS files directly to a z/OS mainframe.
+* Enables you to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
+* Enables you to create, modify, rename and upload USS files directly to a z/OS mainframe.
 * Provides a more streamlined way to access data sets, uss files and jobs.
-* Lets you create and use Zowe CLI `zosmf` compatible profiles.
+* Lets you create, edit and delete Zowe CLI `zosmf` compatible profiles.
+* Lets you use Secure Credential Store plug-in to ensure your credentials security.
```
Comment (zdmullen) [inline README.md:14]: Lets you use the Secure Credential Store plug-in to store your credentials securely in the settings.

File: README.md [hunk 3]
```diff
@@ -60,6 +61,55 @@ Ensure that you meet the following prerequisites before using the extension:
 
 You successfully created a Zowe CLI `zosmf` profile. Now you can use all the functionalities of the extension.
 
+## Secure Credential Store
+
+You can now use Secure Credential Store plug-in in the Zowe Explorer extension. Secure Credentials Store plug-in lets you store your credentials securely. You can use the plug-in in the following ways:
```
Comment (zdmullen) [inline README.md:66]: Use the Secure Credential Store Plug-in for Zowe CLI to store your credentials securely in Zowe Explorer in the settings file.

File: README.md [hunk 4]
```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
 
 The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes. Working with data sets and USS files from VSC can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
 
-* Enables developers to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
-* Enables developers to create, modify, rename and upload USS files directly to a z/OS mainframe.
+* Enables you to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
```
Comment (zdmullen) [inline README.md:10]: I prefer the Oxford comma-- Enables you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.

File: README.md [hunk 5]
```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
...
+* Enables you to create, modify, rename and upload USS files directly to a z/OS mainframe.
```
Comment (zdmullen) [inline README.md:11]: Same here: ...rename, and upload...

File: README.md [hunk 6]
```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
...
 * Provides a more streamlined way to access data sets, uss files and jobs.
-* Lets you create and use Zowe CLI `zosmf` compatible profiles.
+* Lets you create, edit and delete Zowe CLI `zosmf` compatible profiles.
```
Comment (zdmullen) [inline README.md:13]: ...edit, and delete

File: README.md (hunk 3 above)
Comment (IgorCATech) [reply README.md:66]: That covers only 1 way of using the plug-in. The other one is SCS in ZE only (excluding CLI). If I replace what I have with what you suggest, I'll have to edit the rest of the doc and exclude the other way of using the plug-in.

Comment (zdmullen) [reply README.md:66]: Okay, you can leave it for now. I guess my point is that the user won't really see either use case.

**Comment by zFernand0** [comment]
```
README.md: 101: MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]

README.md: 105: MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/2/3]

README.md: 106: MD029/ol-prefix Ordered list item prefix [Expected: 2; Actual: 3; Style: 1/2/3]

README.md: 107: MD029/ol-prefix Ordered list item prefix [Expected: 3; Actual: 4; Style: 1/2/3]

README.md: 108: MD029/ol-prefix Ordered list item prefix [Expected: 4; Actual: 5; Style: 1/2/3]

README.md: 109: MD029/ol-prefix Ordered list item prefix [Expected: 5; Actual: 6; Style: 1/2/3]
```

**Comment by IgorCATech** [comment]
The OL is fixed. What else...

**Comment by jellypuno** [comment]
README.md: 103: MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "   ```"]

File: README.md [hunk 7]
```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
...
+* Lets you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
+* Lets you use the Secure Credential Store plug-in to store your credentials securely in the settings.
 
 **Note:** The Zowe Explorer is powered by [Zowe CLI](https://zowe.org/home/). The extension demonstrates the potential for plug-ins powered by Zowe.
 
 **Tips:**
 
 * For information about how to install the extension from a `VSIX` file and run system tests on the extension, see the [Developer README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README.md).
 * For information about how to develop for Eclipse Theia, see [Theia README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md).
-* For information about how to create a VSCode extension for Zowe Explorer (a Zowe Explorer extension) see [VSCode extensions for Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Extending.md).
```
Comment (zFernand0) [inline README.md:21]: Hey @IgorCATech  and @Colin-Stone,
Should this be added back?

Comment (IgorCATech) [reply README.md]: yes, apparently it was added after I copied the md file and worked on it locally, and then just copied the entire file with changes back. Sorry, my bad.
Good catch, @zFernand0 !

**Comment by zFernand0** [comment]
this PR was already approved : )

**Pull request merged**